pure-variants/setup_workspace_purevariants.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import re
+
+logging.basicConfig(level=logging.DEBUG)
+LOGGER = logging.getLogger("pure::variants")

```

Comment:
```suggestion
LOGGER = logging.getLogger("pure::variants preparation")
```

Reply:
This comes from when I tried to isolate the function in its own file for reusability. Fixed.